configure.ac

```diff
@@ -383,6 +383,56 @@ AC_ARG_ENABLE(openmp, AS_HELP_STRING([--enable-openmp],[Compile with OpenMP thre
   CXXFLAGS="$CXXFLAGS $OPENMP_CXXFLAGS"
   fi
   ])
+
```

Comment:
Let's move this to its own m4 module. Configure AC is already a bit bloated. See the fastFFT submodule for a general idea.